src/libraries/System.Drawing.Common/ref/System.Drawing.Common.cs

```diff
@@ -2118,7 +2117,6 @@ public sealed partial class EncoderParameter : System.IDisposable
         public EncoderParameter(System.Drawing.Imaging.Encoder encoder, short[] value) { }
         public EncoderParameter(System.Drawing.Imaging.Encoder encoder, int numberValues, System.Drawing.Imaging.EncoderParameterValueType type, System.IntPtr value) { }
         public EncoderParameter(System.Drawing.Imaging.Encoder encoder, int numerator, int denominator) { }
-        [System.ObsoleteAttribute("This constructor has been deprecated. Use EncoderParameter(Encoder encoder, int numberValues, EncoderParameterValueType type, IntPtr value) instead.  https://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
This shouldn't be removed. Unfortunately, we don't preserve these attributes in the source sometimes, so they get dropped when running `GenerateReferenceSource` so you have to revert that change.